shared/desktop/app/node.desktop.tsx

```diff
@@ -164,6 +164,9 @@ const willFinishLaunching = () => {
       mainWindowDispatch(DeeplinksGen.createLink({link}))
     }
   })
+  Electron.app.on('open-file', (_, path) => {
+    mainWindowDispatch(DeeplinksGen.createHandleFile({path}))
```

Comment:
@cjb Quick aside. It seems like this needs `event.preventDefault()` to work correctly: https://www.electronjs.org/docs/api/app#event-open-url-macos